tests/ui/requirements.txt

```diff
@@ -1,9 +0,0 @@
-fxapom==1.9.1
-PyJWT==1.4.2
-PyPOM==1.1.1
-pytest==3.0.6
-pytest-instafail==0.3.0
-pytest-selenium==1.8.0
-pytest-variables==1.4
-pytest-xdist==1.15.0
-selenium==3.0.2
```

Comment:
let's keep this and install it separately instead of introducing a mixed requirements file as I mentioned above.